Intermittent test failure in experimenter/reporting/tests/test_tasks.py::TestTask::test_is_duplicate_recipe_change_returns_true

Working on PR #5990 with only a frontend test change, I saw [a python test failure](https://app.circleci.com/pipelines/github/mozilla/experimenter/10390/workflows/d1810136-81a5-4901-9db0-cc1dc76d6683/jobs/30670) that resolved on a re-run.

```
Required test coverage of 100.0% reached. Total coverage: 100.00%
=========================== short test summary info ============================
FAILED experimenter/reporting/tests/test_tasks.py::TestTask::test_is_duplicate_recipe_change_returns_true
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-1513)

--- a/app/experimenter/reporting/tests/test_tasks.py
+++ b/app/experimenter/reporting/tests/test_tasks.py
@@ -145,8 +145,9 @@ class TestTask(MockNormandyMixin, TestCase):
 
     def test_is_duplicate_recipe_change_returns_true(self):
         experiment = ExperimentFactory.create()
+        timestamp = timezone.datetime(2021, 8, 18, 8, 15, 12)
         report_log = ReportLogFactory.create(
-            timestamp=timezone.now(),
+            timestamp=timestamp,
             experiment_type="Normandy-Pref",
             experiment_name=experiment.name,
             experiment_old_status=Experiment.STATUS_LIVE,
@@ -155,7 +156,7 @@ class TestTask(MockNormandyMixin, TestCase):
             event_reason=ReportLogConstants.EventReason.RECIPE_CHANGE,
         )
 
-        later = timezone.now() + timezone.timedelta(hours=1)
+        later = timestamp + timezone.timedelta(hours=1)
         self.assertTrue(
             is_duplicate_recipe_change(later, experiment, report_log.event_reason)
         )

